lib/oracle.js

```diff
@@ -233,10 +294,12 @@ Oracle.prototype.buildInsertReturning = function(model, data, options) {
   var modelDef = this.getModelDefinition(model);
   var type = modelDef.properties[this.idName(model)].type;
   var outParam = null;
-  if (type === String) {
-    outParam = new this.driver.OutParam(this.driver.OCCISTRING);
+  if (type === Number) {
```

Comment:
From my understanding, there is a refactor of `Oracle.prototype.buildInsertReturning` here to support "Bind Parameters", but it seems we don't have test case for this function. Shall we add one? 
I can add it, and would appreciate your suggestion like what's the proper scenario to test it.